flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -110,10 +114,40 @@ function post_test_validation {
     fi
 }
 
+# returns the number of allocated ports
+function get_num_ports {
+    # "ps --ppid 2 -p 2 --deselect" shows all non-kernel processes
+    # "ps --ppid $$" shows all children of this bash process
+    # "ps -o pid= -o comm=" removes the header line
+    echo $(sudo netstat -tulpn | wc -l)
```

Comment:
why do we have to run sudo here? Isn't the test executed by `root`? 🤔

Reply:
afaik the tests are run by `vsts` user on Azure. I just want to be sure that we are catching all ports